packages/cardano-services/src/StakePool/TypeormStakePoolProvider/mappers.ts

```diff
@@ -0,0 +1,116 @@
+import { Cardano, Paginated } from '@cardano-sdk/core';
+import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
+import { RelayModel, mapRelay } from '../DbSyncStakePoolProvider';
+
+export type Margin = {
+  numerator: number;
+  denominator: number;
+};
+
+export type PoolModel = {
+  pool_id: string;
+  pool_status: string;
+  params_pledge: string;
+  params_cost: string;
+  params_margin: Margin;
+  params_relays: RelayModel[];
+  params_owners: string[];
+  params_vrf: string;
+  params_reward_account: string;
+  params_metadata_url: string | null;
+  params_metadata_hash: string | null;
+  metadata_ticker: string | null;
+  metadata_name: string | null;
+  metadata_homepage: string | null;
+  metadata_description: string | null;
+  metadata_ext: JSON | null;
+  total_count: string;
+};
+
+export type PoolStatsModel = {
+  status: string;
+  count: number;
+};
+
+export type PoolStats = {
+  activating: number;
+  active: number;
+  retiring: number;
+  retired: number;
+};
+
+export type SortOrder = 'ASC' | 'DESC' | undefined;
+
+const mapMetadataJson = ({
+  params_metadata_url,
+  params_metadata_hash
+}: PoolModel): Cardano.PoolMetadataJson | undefined => {
+  if (params_metadata_url && params_metadata_hash) {
+    return {
+      hash: params_metadata_hash as Hash32ByteBase16,
+      url: params_metadata_url
+    };
+  }
+};
+
+const mapMetadata = ({
+  metadata_name,
+  metadata_ticker,
+  metadata_homepage,
+  metadata_description
+}: PoolModel): Cardano.StakePoolMetadata | undefined => {
+  if (metadata_name && metadata_ticker && metadata_description && metadata_homepage) {
+    return {
+      description: metadata_description,
+      homepage: metadata_homepage,
+      name: metadata_name,
+      ticker: metadata_ticker
+    };
+  }
+};
+
+// TODO: remove the static metrics once metrics projection is implemented
+const poolMetricsStaticData: Cardano.StakePoolMetrics = {
+  apy: Cardano.Percent(0.015),
```

Comment:
Wouldn't be better to set it as `null` (or `undefined`) since it is actually not currently supported?

Reply:
d7466f611a14c1f889af2aa5992409239122e1ec